example/lib/main.dart

```diff
@@ -39,21 +39,21 @@ class _HomeState extends State<Home> {
 
     Future.delayed(Duration.zero, () {
       if (statusCode == 412) {
-        NativeUpdater.displayUpdateAlert(
-          context,
-          forceUpdate: true,
-          appStoreUrl: '<Your App Store URL>',
-          playStoreUrl: '<Your Play Store URL>',
-          iOSDescription: '<Your iOS description>',
-          iOSUpdateButtonLabel: 'Upgrade',
-          iOSCloseButtonLabel: 'Exit',
+        NativeUpdater.displayUpdateAlert(context,
```

Comment:
context needs to be in a new line, please format the code to keep consistency.

Reply:
my vscode keeps formatting some lines with this strange behavior, this i can fix...